src/components/file-input/examples/file-input.tsx

```diff
@@ -0,0 +1,80 @@
+import { FileInfo } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Basic example
+ */
+@Component({
+    tag: 'limel-example-file-input',
+    shadow: true,
+})
+export class FileInputExample {
+    @State()
+    private disabled = false;
+
+    @State()
+    private readonly = false;
+
+    @State()
+    private multiple = false;
+
+    @State()
+    private files: FileInfo[] = [];
+
+    public render() {
+        return [
+            <limel-file-input
+                onFilesSelected={this.handleFilesSelected}
+                disabled={this.disabled || this.readonly}
+                multiple={this.multiple}
+            >
+                <limel-button label="Select a file" />
+                {this.files.map((file) => (
+                    <limel-chip
+                        text={file.filename}
+                        icon={file.icon}
+                        disabled={this.disabled}
+                        readonly={this.readonly}
+                    />
+                ))}
+            </limel-file-input>,
+            <limel-example-controls>
+                <limel-checkbox
+                    checked={this.disabled}
+                    label="Disabled"
+                    onChange={this.setDisabled}
+                />
+                <limel-checkbox
+                    checked={this.readonly}
+                    label="Readonly"
+                    onChange={this.setReadonly}
+                />
+                <limel-checkbox
+                    checked={this.readonly}
+                    label="Multiple"
+                    onChange={this.setMultiple}
+                />
+            </limel-example-controls>,
+            <limel-example-value value={this.files} />,
+        ];
+    }
+
+    private handleFilesSelected = (event: CustomEvent<FileInfo[]>) => {
+        this.files = [...this.files.concat(event.detail)];
+    };
+
+    private setDisabled = (event: CustomEvent<boolean>) => {
+        event.stopPropagation();
+        this.disabled = !!event.detail;
```

Comment:
Similar point here to the other example

Reply:
See https://github.com/Lundalogik/lime-elements/pull/2819/commits/928bab85158134be9543b283bdd389ba8bc1b83c